pom.xml

```diff
@@ -25,6 +25,7 @@
 	</properties>
 	<modules>
 		<module>spring-cloud-stream</module>
+		<module>spring-cloud-stream-rxjava</module>
```

Comment:
Agreed. It should be in the main repo, but it does need to evolve so that Observable et al are bindable in the same way as MessageChannels.